src/main/java/frc/robot/RobotContainer.java

```diff
@@ -217,7 +227,10 @@ private void configureButtonBindings() {
     m_coDriverController.rightBumper().onTrue(simpleScoreFar);
     m_coDriverController.y().whileTrue(new ScoreReset());
     m_coDriverController.x().whileTrue(prepAmp);
-    m_coDriverController.b().whileTrue(new Puke()).onFalse(new ResetWheels());
+    m_coDriverController
+        .b()
+        .whileTrue(
+            prepFeed); // m_coDriverController.b().whileTrue(new Puke()).onFalse(new ResetWheels());
```

Comment:
```suggestion
            prepFeed);
```

We don't need comments, we have git history for this. But if you want to keep it make sure it's on a different line.